pkg/buses/reconciler.go

```diff
@@ -554,6 +561,7 @@ func (r *Reconciler) createOrUpdateClusterBus(clusterBus *channelsv1alpha1.Clust
 	if r.bus.GetObjectKind().GroupVersionKind().Kind != clusterBus.Kind ||
 		clusterBus.Name != r.bus.GetObjectMeta().GetName() {
 		// this is not our clusterbus
+		r.logger.Infof("Ignoring update for bus (%v/%v) that is not ours.", clusterBus.Namespace, clusterBus.Name)
```

Comment:
ClusterBuses don't have a namespace.

I've been trying to normalize logging references to resources by using the [respective reference](https://github.com/knative/eventing/blob/e6fb7df8f1b83b6e78f6639f7f7461ee75a9b196/pkg/buses/references.go#L49).

`%s` is more idiomatic for string values.